cdk-sqs-eventbridge-pipes-stepfunctions--filter-transform-python/example-pattern.json

```diff
@@ -0,0 +1,54 @@
+{
+  "title": "EventBridge Pipe from SQS to Step Functions with filter and transformation",
+  "description": "Amazon EventBridge Pipe apply filter and transformation from Amazon Simple Queue Service to AWS Step Functions.",
+  "language": "Python",
+  "level": "300",
+  "framework": "CDK",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "The example will create an Amazon Simple Queue Service, AWS Step Functions and Amazon EventBridge Pipe.",
+      "Sending messages to the Amazon Simple Queue Service will trigger the pipe to initiate AWS Step Function execution.",
+      "SQS Messages will be filtered and transformed before sending to AWS Step Functions."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/cdk-sqs-eventbridge-pipes-stepfunctions--filter-transform-python",
+      "templateURL": "serverless-patterns/cdk-sqs-eventbridge-pipes-stepfunctions-filter-transform-python",
+      "projectFolder": "cdk_eventbridge_pipes_sqs_to_stepfunctions_python",
+      "templateFile": "cdk_eventbridge_pipes_sqs_to_stepfunctions_python_stack.py"
+    }
```

Comment:
Can you please take a look at the `projectFolder` and `templateFile` references? They seem to be incorrect.